tests/zfs-tests/tests/functional/mv_files/random_creation.ksh

```diff
@@ -0,0 +1,47 @@
+#!/bin/ksh -p
+
+source "${STF_SUITE}/include/libtest.shlib"
+source "${STF_SUITE}/tests/functional/mv_files/mv_files.cfg"
+
+log_assert "Check that creating many files quickly is safe"
+
```

Comment:
Can you add a comment like "This will test the https://github.com/zfsonlinux/zfs/issues/7401 regression."